src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -442,8 +442,7 @@ def _get_service_principal_object_from_type(servicePrincipalType):
     servicePrincipalResult = None
 
     if (servicePrincipalType is not None and
-        (servicePrincipalType == ServicePrincipalType.system_assigned.value or
-         servicePrincipalType == ServicePrincipalType.none.value)):
+        (servicePrincipalType in (ServicePrincipalType.system_assigned.value, ServicePrincipalType.none.value))):
```

Comment:
This breaks flake8:

```
Running flake8 on modules...
ERROR: /mnt/vss/_work/1/s/src/azure-cli/azure/cli/command_modules/sql/custom.py:445:9: E129 visually indented line with same indent as next logical line
/mnt/vss/_work/1/s/src/azure-cli/azure/cli/command_modules/sql/custom.py:445:9: E129 visually indented line with same indent as next logical line
2     E129 visually indented line with same indent as next logical line
```

See https://stackoverflow.com/questions/44638211/pep8-contradiction-between-e129-and-e127-e128